source/plugins/cloudstack-kubernetes-service.rst

```diff
@@ -92,6 +95,9 @@ Once the ISO has been built for a desired Kubernetes version, it can be added by
 
 |cks-add-version-form.png|
 
+.. note::
+   Since 4.21.0 it is possible to deploy separate dedicated etcd nodes. This requires the Kubernetes ISO contains the etcd binaries.
```

Comment:
```suggestion
   Since 4.21.0 it is possible to deploy separate dedicated etcd nodes. This requires
   the Kubernetes ISO to contains the etcd binaries.
```